tasks/run.json

```diff
@@ -9,6 +9,10 @@
     "environment": {
       "description": "The desired puppet code environment to use",
       "type": "Optional[String[1]]"
+    },
+    "tags": {
+      "description": "Optional string that will be passed to --tags",
+      "type": "Optional[String[1]]"
```

Comment:
Puppet accepts a [comma delimited list of tags](https://github.com/puppetlabs/puppet/blob/1fba2505e6fd5c7ed17257c07667da98d6ed1dfb/lib/puppet/defaults.rb#L2084-L2086), so I'm thinking this should accept an array of non-empty strings

```ruby
"type": "Optional[Array[String[1]]]"
```

And then in the `run.rb` task below, join the tags

```ruby
"--tags=#{params['tags'].join(',')}"